### ISSUE TITLE
All downloads are incomplete..

### ISSUE THREAD

**Issue by dru8274** [rg3/youtube-dl#4840]
I use youtube-dl often, and almost never have a problem. Today, EVERY youtube video I try to download stops early, incomplete, after only fetching maybe 10% of the VOD. It looks like youtube-dl is attempting to download, but the bandwidth is throttled, either from youtube or my ISP (unlikely).

Youtube-dl is a great tool, so thanks be unto the devs. That said, I see no telltale error, so this problem is frustrating. 

A typical output from one of my interrupted downloads...

$ ~/bin/youtube_dl.py --verbose --continue --retries=10 --format=22 http://www.youtube.com/watch?v=1_-xGg9rNKI&feature=youtube_gdata
[1] 11145
$ [debug] System config: []
[debug] User config: []
[debug] Command-line args: ['--verbose', '--continue', '--retries=10', '--format=22', 'http://www.youtube.com/watch?v=1_-xGg9rNKI']
[debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2015.02.02
[debug] Python version 2.7.8 - Linux-3.16.0-4-686-pae-i686-with-debian-8.0
[debug] exe versions: avconv 2.5.3, avprobe 2.5.3, ffmpeg 2.5.3, ffprobe 2.5.3
[debug] Proxy map: {}
[youtube] 1_-xGg9rNKI: Downloading webpage
[youtube] 1_-xGg9rNKI: Extracting video information
[youtube] 1_-xGg9rNKI: Downloading DASH manifest
[debug] Invoking downloader on u'https://r4---sn-j5aa2a0n-53al.googlevideo.com/videoplayback?expire=1422872341&initcwndbps=1200000&ipbits=0&ratebypass=yes&sparams=dur%2Cid%2Cinitcwndbps%2Cip%2Cipbits%2Citag%2Cmime%2Cmm%2Cms%2Cmv%2Cpl%2Cratebypass%2Crequiressl%2Csource%2Cupn%2Cexpire&mime=video%2Fmp4&upn=oZCB_Oh54qw&sver=3&fexp=900718%2C904846%2C907263%2C927622%2C930676%2C931344%2C934954%2C936118%2C9405889%2C943917%2C947225%2C948124%2C948701%2C948703%2C952302%2C952605%2C952809%2C952901%2C955301%2C957201%2C958101%2C959701&signature=87814EF9288F1FA6A7E154B3B0AF0DBA0793E5D5.A70F1215F65F208AB6003060D454A751C380AF2A&mt=1422850681&mv=m&pl=17&ms=au&mm=31&key=yt5&source=youtube&dur=5708.323&id=o-ANYSjv-YaZXMeV_cyCh61f0XProqmg_D6UUhAnNr2zxh&itag=22&requiressl=yes&ip=121.99.25.249'
[download] Destination: StarCraft 2 - Maru vs. herO (TvP) - IEM 2015 Taipei - Quarterfinal-1_-xGg9rNKI.mp4
[download] 100% of 1.48GiB in 02:01
ERROR: content too short (expected 1585219651 bytes and served 312474013)
Traceback (most recent call last):
  File "/home/nostromo/bin/youtube_dl.py/youtube_dl/YoutubeDL.py", line 1283, in process_info
    success = dl(filename, info_dict)
  File "/home/nostromo/bin/youtube_dl.py/youtube_dl/YoutubeDL.py", line 1258, in dl
    return fd.download(name, info)
  File "/home/nostromo/bin/youtube_dl.py/youtube_dl/downloader/common.py", line 316, in download
    return self.real_download(filename, info_dict)
  File "/home/nostromo/bin/youtube_dl.py/youtube_dl/downloader/http.py", line 218, in real_download
    raise ContentTooShortError(byte_counter, int(data_len))
ContentTooShortError

[1]+  Exit 1                  ~/bin/youtube_dl.py --verbose --continue --retries=10 --format=22 http://www.youtube.com/watch?v=1_-xGg9rNKI

**Comment by dru8274** [comment]
The above problem is still happening. But it does not happen with all videos as I said. Rather, it happens mainly with extra large youtube videos of 600Mbyte+. Typically where I have attempted to download with --format=22. In which case, the download halts after perhaps just 10% or 20% completed.

Perhaps it has always been like this - I do not know. But either way, I have avoided the problem for now by downloading the affected videos in lower quality, and restarting the download multiple times as necessary until completed. 

In short, this issue has been resolved for now through problem avoidance.
